data/latest/run-all-tests-locally.sh

```diff
@@ -112,9 +117,13 @@ snap_remove 2>/dev/null > /dev/null
 set +e
 if [ -n "$SINGLE_TEST" ]; then
     printf "running single test: %s ..." "$SINGLE_TEST"
+    if [ "$SINGLE_TEST" == "test-refresh-services.sh" ]; then
+        snap_download_stable_and_latest
+    fi
 
-    if [ "$SINGLE_TEST" == "test-refresh-config-paths.sh" ] ||
-       [ "$SINGLE_TEST" == "test-refresh-services.sh" ]; then
+    if [ "$SINGLE_TEST" == "test-refresh-config-paths.sh" ]; then
+        EDGEX_IRELAND_SNAP_FILE=$(snap_download_and_ack edgexfoundry --channel=2.0/stable)
+        export EDGEX_IRELAND_SNAP_FILE
```

Comment:
Call this `EDGEX_PREV_STABLE_SNAP_FILE` to simplify future updates?

Reply:
Thanks! Updated.